collectors/bzimport/convertors.py

```diff
@@ -76,6 +76,9 @@ def groups_read(self):
         """
         appropriate read LDAP groups
         """
+        if self.bug.get("groups", []) == ["redhat"]:
```

Comment:
The whole branching logic should be re-thinked a bit more. Assume that for some reason the groups are `["redhat", "devel"]`. Then, we would endup with a public flaw in OSIDB. Despite encountering unexpected group we should not extend the visibility about Bugzilla.

I suggest to make the logic in a way that `security` group present will always make it embargoed, otherwise any other group present (including `redhat` but also any other unexpected group) will make it internal, no groups will make it public.